ccmbioinfo/lfs
# CARDS-2098: PREMS: Add support for importing only one visit per event

To test:
- build with `-Pdocker`
- in `compose-cluster` run `python3 generate_compose_yaml.py --mssql --cards_project cards4prems --oak_filesystem --dev_docker_image --smtps --smtps_test_container --smtps_test_mail_path ~/cardsmail --composum --adminer`
- start with `docker-compose build && docker-compose up`
- at `http://localhost:1435` log in and import `compose-cluster/mssql/processorImportTestSql.sql`
- trigger an import at `localhost:8080/Subjects.importClarity?pastDayToQuery=0` (adjust with the number of days since 2023-02-20), wait a bit, make sure that patient `0000008` has only one visit imported
